src/language/syntax/rules/profile/profile.ts

```diff
@@ -99,19 +100,21 @@ export const ENUM_VALUE: SyntaxRule<WithLocation<EnumValueNode>> =
       '}',
       '\n'
     ).map(([name, maybeAssignment]): WithLocation<EnumValueNode> => {
-      let enumValue: string | number | boolean;
+      const variantName: string = name.data.identifier;
+      let variantValue: string | number | boolean;
+
       if (maybeAssignment === undefined) {
-        enumValue = name.data.identifier;
```

Comment:
How it worked before? From this line it seems like, the variant value before wasn't used at all. 🤔

Reply:
This is under the `maybeAssignment === undefined` if. The `maybeAssignment` rule optionally matches `= {literal}`, so if it wasn't present, we'd use the enum variant name as the string value.

Reply:
Ok, I was blind, thanks for explanation.